src/Core/Scheduler/Providers/PointCloudProvider.js

```diff
@@ -83,7 +83,15 @@ function parseOctree(layer, hierarchyStepSize, root) {
                         const myname = childname.substr(root.name.length);
                         url = `${root.baseurl}/${myname}`;
                     }
-                    const item = { numPoints: n, childrenBitField: c, children: [], name: childname, baseurl: url, bbox: bounds };
+                    const item = {
+                        numPoints: n,
+                        childrenBitField: c,
+                        children: [],
+                        name: childname,
+                        baseurl: url,
+                        bbox: bounds,
+                        geometricError: layer.metadata.spacing / Math.pow(2, childname.length),
```

Comment:
Evoked in the commit message, there is a reference that explains the calculation?

Reply:
This is by construction: see https://github.com/peppsac/PotreeConverter/blob/master/PotreeConverter/src/PotreeWriter.cpp#L73